src/runner.py

```diff
@@ -0,0 +1,35 @@
+""" Function runner can  de called as:
+runner() – print results of all function
+runner(‘generate_numbers’) – print result  only for generate_numbers()
+runner(‘generate_numbers’, ‘happy_numbers’) – print results for gen_numbers()
+and happy_numbers(). Any combination of functions can be specified."""
+
+import pytasks
+from random import choice
+
+
+def runner(*funcs):
+    module_funcs = dir(pytasks)
+    module_funcs = [func for func in module_funcs if not func.startswith('__')]
+    if not funcs:
+        for func in module_funcs:
```

Comment:
what would happen if func is not a function? i.e. i've created variable in module